pkg/providers/instancetype/types.go

```diff
@@ -132,6 +132,13 @@ func computeRequirements(ctx context.Context, info *ec2.InstanceTypeInfo, offeri
 	return requirements
 }
 
+func getOS(info *ec2.InstanceTypeInfo, amiFamily amifamily.AMIFamily) string {
+	if _, ok := amiFamily.(*amifamily.Windows); ok && getArchitecture(info) == v1alpha5.ArchitectureAmd64 {
```

Comment:
Does this make sense? i.e. If the family is Windows, I think we should return:
1. Windows if the architecture is `amd64`
2. Empty string (no value) if it's anything else (meaning that windows doesn't support it)
In the case that the family isn't windows, then we always return `linux`

Reply:
Yours is more accurate. :)